Sources/Shared/API/Responses/Logbook.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import ObjectMapper
+
+public struct LogbookResponse: ImmutableMappable {
```

Comment:
```suggestion
public struct LogbookEntry: ImmutableMappable {
```

Along with renaming the file to match, I think this is a little more clear - `LogbookResponse` would probably be a thing which contains the array, not the items inside the array.

Reply:
Done in 3a3e23cc374738d5c6ba1124f93f02f55f6ba1f8